pkgs/development/libraries/libargon2/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "libargon2-${version}";
+  version = "20161029";
+  src = fetchFromGitHub {
+    owner = "P-H-C";
+    repo = "phc-winner-argon2";
+    rev = "${version}";
+    sha256 = "021g8wi4g67ywm8zf3yncqwrmfz7ypgm1ih9wcmnxip5n75rymh5";
+  };
+
+  patches = [ ./pkgconfig.patch ];
+
+  installPhase = ''
+    mkdir -p $out
+    make install PREFIX=$out
+    mkdir -p $out/lib/pkgconfig
+    cat <(echo -e "prefix=$out\nversion=${version}") libargon2.pc > $out/lib/pkgconfig/libargon2.pc
```

Comment:
Working on it. I'm not very familiar with pkgconfig though, and I don't see other packages dealing with this at all, so I'm not sure what to do.